src/transport/SecureSessionMgr.h

```diff
@@ -44,6 +44,23 @@ namespace chip {
 
 class SecureSessionMgr;
 
+
+class SecureSessionHandle
+{
+public:
+    SecureSessionHandle() : mPeerNodeId(kAnyNodeId), mPeerKeyId(0) {}
+    SecureSessionHandle(NodeId peerNodeId, uint16_t peerKeyId) : mPeerNodeId(peerNodeId), mPeerKeyId(peerKeyId) {}
```

Comment:
Should this constructor be protected, so only `SecureSessionMgr` can use it?

Reply:
In the future, it is. But for now, InteractionModel need to create it using this one.